simple_config

```diff
@@ -2,6 +2,8 @@
 
 port 2020;  # This is also a comment.
 
+numThreads 5;
```

Comment:
I thought we agreed on `snake_case` for this param during our offline discussion? Was there a later update/decision I missed?